source/agora/utils/InetUtils.d

```diff
@@ -41,11 +41,24 @@ public enum HostType : ubyte
     Domain,
 }
 
-///
-mixin template AddGetAllIPs()
+struct InetUtils
 {
-    public static string[] getAllIPs()
+    version (Posix) public static string[] getAllIPs()
     {
+        import core.sys.posix.netdb;
+
+        version (OSX)
+        {
+            import core.sys.darwin.ifaddrs;
+            import core.sys.posix.netinet.in_;
+            import core.sys.posix.sys.socket;
+        }
+        version (linux)
+        {
+            import core.sys.linux.ifaddrs;
+            import core.sys.posix.netdb;
```

Comment:
Can this be removed due to the same import in L48?

Reply:
Added a commit.